src/main/java/org/jenkinsci/test/acceptance/po/DumbSlave.java

```diff
@@ -48,9 +50,14 @@ public void asLocal() {
         File jar = new File(tmpDir, "slave"+createRandomName()+".jar");
         String command;
         if (SystemUtils.IS_OS_UNIX) {
+            String java = "java";
+            String jh = System.getProperty("java.home");
+            if (!StringUtils.isEmpty(jh)) {
+                java = new File(jh, "bin/java").getAbsolutePath();
```

Comment:
Huh? I needed the full path to java because my PATH env var was not available when I run from within the IDE, it still worked when running mvn from commandline, but this was needed to debug easier.